ext/civiimport/Managed/ImportSearches.mgd.php

```diff
@@ -0,0 +1,182 @@
+<?php
+
+use Civi\Api4\Entity;
+use Civi\BAO\Import;
+use CRM_Civiimport_ExtensionUtil as E;
+
+// Check if SearchKit is enabled before adding SavedSearches.
+try {
+  if (!Entity::get(FALSE)
+    ->addWhere('name', '=', 'SavedSearch')
```

Comment:
`SavedSearch` is actually a core entity.
```suggestion
    ->addWhere('name', '=', 'SearchDisplay')
```

Reply:
Also, I really think we ought to teach `ManagedEntities` to gracefully skip non-existent entities.

Reply:
Fixed - also pushed a couple more tweaks